src/components/Home.js

```diff
@@ -0,0 +1,130 @@
+import { useState } from "react";
+import { Icon } from "@iconify/react";
+import styled from "styled-components";
+
+import Modal from "./common/Modal";
+import photo from "../assets/tree.png";
+
+export default function Home() {
+  const welcomeMessage =
+    "주변에 말 못할 고민이 있으신가요? 익명의 사람들과 고민을 나눠보세요!";
+  const buttonName = "Discover Now";
+  const copyright = "All rights reserved to anon-wall";
+
+  const [modalOn, setModalOn] = useState(false);
+
+  const handleModalOn = () => {
+    setModalOn(true);
+  };
+
+  return (
+    <>
+      <UpperBoxWrapper />
+      <MiddleContainer>
+        <DescriptionContainer>
+          <div className="description">
+            <div className="message">{welcomeMessage}</div>
+            <button onClick={handleModalOn}>{buttonName}</button>
+            {modalOn && (
+              <Modal onClick={handleModalOn} width="300px" height="600px" />
```

Comment:
```suggestion
              <Modal onClick={setModalOn} width="300px" height="600px" />
```
useState 의 Setter 함수를 넣고 테스트 한 번 해보세요!